### PR TITLE
JavaScript - Flow Control

### PR DESCRIPTION
**Pull request by kapnobatai137**
- discussion insight

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/flow-control/flow-control-discussion-insight.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/flow-control/flow-control-discussion-insight.md?ref=2e9a5066c16e1497260476a7afad6125cc8e4b58)

File: javascript/core/flow-control/flow-control-discussion-insight.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Content
+
+Now that you learned some JavaScript, it's time to apply it!
+
+Let's say you are a farmer, and you need someone to count all the legs of your animals.
+
+You remember that you learned JavaScript with Enki and that you can do this through coding.
+
+How would you write a function that takes as input the number of each animal and returns the total number of legs?
```
Comment (Stefan-Stojanovic) [inline javascript/core/flow-control/flow-control-discussion-insight.md:29]:
```suggestion
How would you write a function that takes as input the number of all animals and returns the total number of legs?
```

Would this maybe sound better?

That "number of each animal" sounds a bit weird imo. I had to re-read it a few times to understand what is meant

Comment (nem035) [reply javascript/core/flow-control/flow-control-discussion-insight.md:29]: Based on the attached repl, the text should also mention the animal name.

How about something like this?

````suggestion
How would you write a function that takes two inputs, the type of an animal and how many of those animals are at the farm, and returns the total number of legs?

For example, in our Enki farm, we count the number of legs for our cows like:

```js
const animal = 'cow';
const numberOfAnimals = 10;

countLegs(animal, numberOfAnimals); // 40
```
````

Comment (nem035) [reply javascript/core/flow-control/flow-control-discussion-insight.md:29]: Can also put the example somewhere around line 37, or in the footnote.

**Review by nem035 — DISMISSED** [review]
:tada:

**Pull request merged**